test/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "bedrock-payment-paypal-test",
+  "version": "0.0.1",
+  "description": "Integration Tests for Bedrock Payment PayPal.",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "test": "node --preserve-symlinks test test"
+  },
+  "keywords": [
```

Comment:
Can simplify and remove the fields that are not needed for tests to run.  The ones in the top level package.json are the ones that matter.

Reply:
While I will remember this in the future... fondly... do I really need to eliminate more identifying information from a package.json?

Reply:
Don't need to, just suggesting how to simplify away unused fields.

Reply:
Going to skip on this one then. I did refactor the errors: https://github.com/digitalbazaar/bedrock-payment-paypal/pull/2/commits/25bcd406c6c49b9ee691a4f748e1367c851ef25e